app/api/slang.py

```diff
@@ -52,7 +74,7 @@ def generate_slang_internal():
             vocabulary=vocab,
             start_letter="random",
             max_len=20,  # TODO: Fix this. (?)
-            temperature=0.3,
+            temperature=0.35,
```

Comment:
And this whole thing starting at line 68 (found_one = False) up until 83 (print(f"Warning...")) could also be refactored into its own function like "generate_non_existent_word" or so. Think it'd be a bit cleaner with those single-purpose methods / functions.